lib/libzpool/zrepl_mgmt.c

```diff
@@ -20,6 +20,51 @@ SLIST_HEAD(, zvol_info_s) stale_zv_list;
 
 static int uzfs_zinfo_free(zvol_info_t *zinfo);
 
+int
+create_and_bind(char *port, int bind_needed)
```

Comment:
const char *port - would be nice so that we can pass const string defined by define here.

Reply:
Done.